cocotb 1.7.0 sdist install fails (_vendor/vpi/sv_vpi_user.h: No such file or directory)

I tried to build cocotb manually, but I fail:

``` 
building 'cocotb/libs/libcocotbvpi_icarus' extension
creating build/temp.linux-x86_64-cpython-310/cocotb/libs/libcocotbvpi_icarus
creating build/temp.linux-x86_64-cpython-310/cocotb/libs/libcocotbvpi_icarus/cocotb
creating build/temp.linux-x86_64-cpython-310/cocotb/libs/libcocotbvpi_icarus/cocotb/share
creating build/temp.linux-x86_64-cpython-310/cocotb/libs/libcocotbvpi_icarus/cocotb/share/lib
creating build/temp.linux-x86_64-cpython-310/cocotb/libs/libcocotbvpi_icarus/cocotb/share/lib/vpi
x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DCOCOTBVPI_EXPORTS= -DVPI_CHECKING=1 -DICARUS= -D__STDC_FORMAT_MACROS= -Icocotb/share/include -Icocotb -I/tmp/build-env-frrfwotg/include -I/usr/include/python3.10 -c cocotb/share/lib/vpi/VpiCbHdl.cpp -o build/temp.linux-x86_64-cpython-310/cocotb/libs/libcocotbvpi_icarus/cocotb/share/lib/vpi/VpiCbHdl.o -std=c++11 -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wcast-qual -Wwrite-strings -Wconversion -Wno-missing-field-initializers -Wnon-virtual-dtor -Woverloaded-virtual -flto
In file included from cocotb/share/lib/vpi/VpiCbHdl.cpp:34:
cocotb/share/lib/vpi/VpiImpl.h:42:10: fatal error: _vendor/vpi/sv_vpi_user.h: No such file or directory
   42 | #include "_vendor/vpi/sv_vpi_user.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1

ERROR Backend subprocess exited when trying to invoke build_wheel
```

I'm on cocotb `v1.7.0` tag in an environment that can build cocotb `v1.6.0` successfully.

OS: Ubuntu 22.04 LTS
Dependencies:
```
$ sudo apt-get install make gcc g++ python3 python3-dev python3-pip
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
g++ is already the newest version (4:11.2.0-1ubuntu1).
gcc is already the newest version (4:11.2.0-1ubuntu1).
make is already the newest version (4.3-4.1build1).
python3 is already the newest version (3.10.4-0ubuntu2).
python3-dev is already the newest version (3.10.4-0ubuntu2).
python3-pip is already the newest version (22.0.2+dfsg-1).
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,4 +1,5 @@
 recursive-include cocotb/share *
+recursive-include cocotb/_vendor *
 include README.md
 include LICENSE
 include cocotb_build_libs.py

